Orange/widgets/data/tests/test_owrank.py

```diff
@@ -238,12 +245,13 @@ def test_data_which_make_scorer_nan(self):
                 [-np.power(10, 10), 1, 1],
                 [0, 1, 1]
             )))
-        self.widget.score_checks[3].setChecked(True) #ANOVA
+        self.widget.selected_methods.add('ANOVA')
         self.send_signal(self.widget.Inputs.data, table)
 
-    def test_setting_migration_extends_header_state(self):
+    def test_setting_migration_fixes_header_state(self):
         # Settings as of version 3.3.5
         settings = {
+            '__version__': 1,
```

Comment:
Settings in 3.3.5 did not have versions yet. Therefore, __version__ was not present in the dict.